tests/Unit/Order/FeedTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace BazaarVoice\Order;
+
+use BazaarVoice\Elements\InteractionElement;
+use BazaarVoice\Elements\ProductElement;
+use PHPUnit\Framework\TestCase;
+
+class FeedTest extends TestCase
+{
+    public function testNewOrderElement()
```

Comment:
what's the testing convention? other tests use `/** @test */` for example.

a nice to have is a CONTRIBUTING,md guide with these kind of recommendations.